docs/docs/development/database/migration-plan.md

```
@@ -0,0 +1,958 @@
+}
+
+enum FileType {
+  PDF
+  CSV
+  KML
+  EXCEL
+  IMAGE
+}
+```
+
+#### 3. NormalizationTaskテーブル
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)